scifight/models.py

```diff
@@ -27,9 +45,11 @@ class Meta:
 
 
 class Team(models.Model):
-    name = models.CharField(max_length=NAME_LENGTH)
+    tournament  = models.ForeignKey(Tournament)
+    name        = models.CharField(max_length=NAME_LENGTH)
+    slug        = models.SlugField(max_length=SLUG_LENGTH, null=True, blank=True, unique=True)
```

Comment:
Does `unique=True` play nice with `null=True`?

Moreover, we do not want team slugs be completely unique, instead we would like `unique_together = ('tournament', 'slug')`. How does it deal with possible `NULL`s?